specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2025-01-01-preview/mfe.json

```diff
@@ -9,6 +9,90 @@
     "https"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/availableQuota/default": {
```

Comment:
>/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/availableQuota/default [](http://example.com/codeflow?start=5&length=98)

You need to add the corresponding collection GET (LIST) API also, even though it will only return a single item. This is required for core ARM features (ARG, Azure Policy).